packages/mercury/src/tokens/primitive/_color.scss

```diff
@@ -195,11 +211,26 @@
     --mer-color__message-green--200: #3fa89b;
     --mer-color__message-yellow--100: #f9bd56;
     --mer-color__message-yellow--200: #f9bd56;
-    --mer-color__message-red--100: #e35861;
-    --mer-color__message-red--200: #d54b54;
-    --mer-color__message-red--300: #ff6161;
-    --mer-color__message-red--400: #ff9999;
-    --mer-color__message-red--500: #800000;
-    --mer-color__message-red--600: #4d0000;
+    --mer-color__message-red--100: #fcf4f4;
```

Comment:
Just in case, remember to update the Globant variant. In the near future, the showcase will allow you to toggle this variant with a single click.

Reply:
I will update various tokens in another PR. Thanks for noticing.